src/main/java/com/openlattice/chronicle/util/ChronicleServerUtil.java

```diff
@@ -0,0 +1,108 @@
+package com.openlattice.chronicle.util;
+
+import com.google.common.base.Preconditions;
+import com.openlattice.chronicle.constants.*;
+import com.openlattice.chronicle.data.FileType;
+import com.openlattice.chronicle.services.enrollment.EnrollmentManager;
+import org.apache.olingo.commons.api.edm.FullQualifiedName;
+import org.springframework.http.MediaType;
+
+import javax.servlet.http.HttpServletResponse;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+import static com.openlattice.chronicle.constants.EdmConstants.PARTICIPANTS_PREFIX;
+import static com.openlattice.chronicle.constants.EdmConstants.PERSON_ID_FQN;
+
+/**
+ * @author alfoncenzioka &lt;alfonce@openlattice.com&gt;
+ */
+public class ChronicleServerUtil {
+
+    public static String getFirstValueOrNull( Map<FullQualifiedName, Set<Object>> entity, FullQualifiedName fqn ) {
+        if ( entity.getOrDefault( fqn, Set.of() ).isEmpty() ) {
+            return null;
+        }
+        return entity.get( fqn ).iterator().next().toString();
+    }
+
+    public static UUID getFirstUUIDOrNull( Map<FullQualifiedName, Set<Object>> entity, FullQualifiedName fqn ) {
+        String firstValue = getFirstValueOrNull( entity, fqn );
+        if ( firstValue == null ) {
+            return null;
+        }
+
+        try {
+            return UUID.fromString( firstValue );
+        } catch ( IllegalArgumentException e ) {
+            return null;
+        }
+    }
+
+    public static void checkNotNullUUIDs( Set<UUID> uuids ) {
+        for ( UUID uuid : uuids ) {
+            Preconditions.checkNotNull( uuid );
+        }
+    }
+
+    public static String getParticipantEntitySetName( UUID studyId ) {
+        return PARTICIPANTS_PREFIX.concat( studyId.toString() );
+    }
```

Comment:
Can you mark this as `@Deprecated` since we won't need this after we move to app configs world?

Reply:
Um, I don't think we need to mark as deprecated since it's not an api handling method